From 98d62c53ae66b1b55dfa4d48a5e5e6a5809eb084 Mon Sep 17 00:00:00 2001
From: "Christensen, Taylor" <taylor.christensen@intel.com>
Date: Mon, 9 Nov 2015 16:22:36 -0800
Subject: [PATCH] SDHCI: Set HW Mutex Flag

Enable the hardware mutex flag so that the host knows
that the HW mutex is enabled in the SDHCI code set.
---
 drivers/mmc/host/sdhci-acpi.c |    4 ++++
 1 file changed, 4 insertions(+)

diff --git a/drivers/mmc/host/sdhci-acpi.c b/drivers/mmc/host/sdhci-acpi.c
index 284b984..5797a7f 100644
--- a/drivers/mmc/host/sdhci-acpi.c
+++ b/drivers/mmc/host/sdhci-acpi.c
@@ -144,6 +144,10 @@ static int sdhci_acpi_probe_slot(struct platform_device *pdev)
 	pm_qos_add_request(host->mmc->qos, PM_QOS_CPU_DMA_LATENCY,
 					PM_QOS_DEFAULT_VALUE);
 
+#ifdef CONFIG_HW_MUTEXES
+	host->flags |= SDHCI_SUPPORT_HW_MUTEX;
+#endif
+
 	return 0;
 }
 
-- 
1.7.9.5

